optaweb-tsp-ui/package.json

```diff
@@ -2,48 +2,41 @@
   "private": true,
   "license": "Apache-2.0",
   "dependencies": {
-    "gh-pages": "^1.2.0",
+    "@patternfly/react-core": "^1.36.0",
     "leaflet": "^1.3.4",
-    "prop-types": "^15.6.2",
-    "react": "^16.6.0",
-    "react-dom": "^16.6.0",
-    "react-leaflet": "^2.1.1",
-    "react-redux": "^5.1.0",
-    "react-scripts": "2.1.1",
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3",
+    "react-leaflet": "^2.1.2",
+    "react-redux": "^5.1.1",
+    "react-scripts-ts": "3.1.0",
     "redux": "^4.0.1",
     "redux-logger": "^3.0.6",
-    "redux-thunk": "^2.3.0",
     "sockjs-client": "^1.3.0",
     "tachyons": "^4.11.1",
-    "webstomp-client": "^1.2.5"
+    "webstomp-client": "^1.2.6"
   },
   "scripts": {
-    "predeploy": "npm run build",
-    "deploy": "gh-pages -d build",
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "lint": "./node_modules/eslint/bin/eslint.js ./src",
-    "lint:fix": "npm run lint -- --fix",
-    "test": "react-scripts test --env=jsdom",
-    "eject": "react-scripts eject"
+    "start": "react-scripts-ts start",
+    "build": "react-scripts-ts build",
+    "test": "react-scripts-ts test --env=jsdom",
+    "eject": "react-scripts-ts eject"
   },
   "devDependencies": {
-    "ajv": "^6.5.2",
+    "@types/enzyme": "^3.1.15",
+    "@types/enzyme-adapter-react-16": "^1.0.3",
+    "@types/jest": "^23.3.9",
+    "@types/leaflet": "^1.2.14",
+    "@types/node": "^10.12.10",
+    "@types/react": "^16.7.7",
+    "@types/react-dom": "^16.0.10",
+    "@types/react-leaflet": "^1.1.6",
+    "@types/react-redux": "^6.0.10",
+    "@types/redux-logger": "^3.0.6",
+    "@types/sockjs-client": "^1.1.0",
     "enzyme": "^3.7.0",
-    "enzyme-adapter-react-16": "^1.6.0",
-    "eslint-config-airbnb": "^17.1.0",
-    "eslint-plugin-import": "^2.14.0",
-    "eslint-plugin-jest": "^21.27.1",
-    "eslint-plugin-jsx-a11y": "^6.1.2",
-    "eslint-plugin-react": "^7.11.1",
-    "react-test-renderer": "^16.6.0",
-    "redux-devtools-extension": "^2.13.5",
-    "redux-mock-store": "^1.5.3"
-  },
-  "browserslist": [
```

Comment:
I've to check if browserlist works with TS compiler (work with a babel-plugin and this can be not applicable to **tsc**)
About dev tools _redux-devtools-extension_ is still in place, redux-mock-store isn't still used.